consensus/core_chainlock_test.go

```diff
@@ -138,7 +138,8 @@ func invalidProposeCoreChainLockFunc(t *testing.T, height int64, round int32, cs
 
 	// Make proposal
 	propBlockID := types.BlockID{Hash: block.Hash(), PartSetHeader: blockParts.Header()}
-	proposal := types.NewProposal(height, cs.state.NextCoreChainLock.CoreBlockHeight-1, round, cs.ValidRound, propBlockID)
+
+	proposal := types.NewProposal(height, 2, round, cs.ValidRound, propBlockID)
```

Comment:
This is wrong, we are trying to propose an invalid core chain lock by having the height of the next one -1, not 2.

Reply:
Got it. Brought it back.